### PR TITLE
test: AIRDROP-15 for hip-904 test plan

### LINKED ISSUE #14445: (AIRDROP - 15) EOA account airdrops a single token to an account alias with NO free association slots (maxAutoAssociations = 1 and 1 auto association is already used) for FT and NFT

### PR DESCRIPTION
**Pull request by ibankov**
**Description**:

Airdrop-15 from hip 904 test plan

**Related issue(s)**:

Fixes #14445

**Notes for reviewer**:

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

File: hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/hip904/TokenAirdropTest.java [hunk 1]
```diff
@@ -422,6 +425,32 @@ final Stream<DynamicTest> consequentAirdrops() {
                             // assert the account balance
                             getAccountBalance(receiver).hasTokenBalance(FUNGIBLE_TOKEN, 10));
         }
+
+        @HapiTest
+        @DisplayName("that is alias with 0 free maxAutoAssociations")
+        final Stream<DynamicTest> autoCreateWithNftFallBackFeeFails() {
```
Comment (david-bakin-sl) [inline hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/hip904/TokenAirdropTest.java:431]: Test name is not right for this test (copy/paste error/)

Also, this seems to be a success test.  The linked issue is ambiguous: is it a success test (as written here) or a failure test (the failure would be: no more free association slots and existing slot used by a _different_ token) - I didn't go and check the test plan (actually I'm not sure where it is).

**Review by david-bakin-sl — COMMENTED** [review]
Not sure what the issue says the test here should be.

File: hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/hip904/TokenAirdropTest.java (hunk 1 above)
Comment (ibankov) [reply hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/hip904/TokenAirdropTest.java]: renamed

**Review by JivkoKelchev — APPROVED** [review]
LGTM, ty @ibankov

**Review by david-bakin-sl — APPROVED** [review]

**Review by Neeharika-Sompalli — APPROVED** [review]

**Pull request merged**